provider/bidengine/order.go

```diff
@@ -434,6 +434,12 @@ func (o *order) shouldBid(group *dtypes.Group) (bool, error) {
 		return false, nil
 	}
 
+	for _, resources := range group.GroupSpec.GetResources() {
+		if len(resources.Resources.Storage) > o.cfg.MaxGroupVolumes {
+			o.log.Debug(fmt.Sprintf("unable to fulfill: group volumes count exceeds (%d > %d)", len(resources.Resources.Storage), o.cfg.MaxGroupVolumes))
```

Comment:
Seems like this could just be `Info` log level? It shouldn't happen too often.

Reply:
indeed